setup.py

```diff
@@ -2,12 +2,20 @@
 
 try:
     from setuptools import setup, Extension
+    from setuptools.command.build_ext import build_ext as base_build_ext
 except ImportError:
-    from distutils.core import setup, Extension
+    import warnings
+    import sys
+    val = sys.exc_info()[1]
 
+    warnings.warn("import of setuptools failed %r" % val)
```

Comment:
I don't think this warning is necessary -- it falls back to distutils anyway. It seems to me this was added to debug a spelling mistake in an earlier commit. Now that we're importing it correctly, let's just leave this out.

Reply:
i'd like to leave it in so user see that setuptools is missing
from a packaging standpoint i'd like to completely remove the support for plain distutils if this was my project